File: MODUL-2/SOAL 5/PRAK205-2410817120017-RIKA-FAULIANA-RAHMI.c
#include <stdio.h>
#include <math.h>

void hitung_segitiga(float a, float b) {
    // Menghitung alas menggunakan teorema Pythagoras
    float c = sqrt(b * b - a * a);
    
    // Menghitung keliling
    float keliling = a + b + c;
    
    // Menghitung luas
    float luas = 0.5 * c * a;
    
    // Menampilkan hasil
    printf("Alas = %.0f cm\n", c);
    printf("Tinggi = %.0f cm\n", a);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);
}

int main() {
    float a, b;
    
    printf("Masukkan panjang A (tinggi): ");
    scanf("%f", &a);
    printf("Masukkan panjang B (sisi miring): ");
    scanf("%f", &b);
    
    // Memanggil fungsi untuk menghitung segitiga
    hitung_segitiga(a, b);

    return 0;
}

File: MODUL-2/SOAL 5/PRAK205-2410817120017-RIKA-FAULIANA-RAHMI.py
import math

def hitung_segitiga(a, b):
    # Menghitung alas menggunakan teorema Pythagoras
    c = int(math.sqrt(b**2 - a**2))
    
    # Menghitung keliling
    keliling = a + b + c
    
    # Menghitung luas
    luas = int(0.5 * c * a)
    
    # Menampilkan hasil tanpa angka desimal
    print(f"Alas = {c} cm")
    print(f"Tinggi = {a} cm")
    print(f"Keliling = {keliling} cm")
    print(f"Luas = {luas} cm^2")

# Input dari pengguna
a = int(input("Masukkan panjang A (tinggi): "))
b = int(input("Masukkan panjang B (sisi miring): "))

# Memanggil fungsi untuk menghitung segitiga
hitung_segitiga(a, b)
